src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -156,17 +156,17 @@ const Contact: React.FC<Props> = ({
                   value={values.tagValue}
                   onChange={(event) => handleChange(event, 'tagValue')}
                 >
-                  {Object.entries(contactTags).map(
-                    ([statusValue, statusTranslated]) => (
+                  {Object.entries(contactTags)
+                    .slice(0, Object.entries(contactTags).length - 3)
```

Comment:
This seems like another magic number. Why are you removing a certain number of them?

Reply:
In the original app the statuses "all hidden, all active, and none" aren't options for this page but are in the adding appeal form. Does it makes more sense to have a separate list for both these pages?

Reply:
Gotcha. Again, I think filtering them out by id/name and loading them from the API would be better.